.github/workflows/CICD.yml

```diff
@@ -11,7 +11,7 @@ env:
   PROJECT_NAME: coreutils
   PROJECT_DESC: "Core universal (cross-platform) utilities"
   PROJECT_AUTH: "uutils"
-  RUST_MIN_SRV: "1.32.0" ## v1.32.0 - minimum version for half, tempfile, etc
+  RUST_MIN_SRV: "1.49.0" ## v1.33.0 - minimum version for aarch64-apple
```

Comment:
This is a huge bump.
I am ok jumping to 33  but 49 is a bit too much :/

Reply:
1.49 is the minimum version for macOS aarch64 support.  I think 1.33 covers the libc bump which is required even if you are using 1.49.

Reply:
Sure but just create a new task to build it for aarch64-apple.
We can keep 32 for the rest of the platform

Reply:
I just bumped it to 1.33 for the libc. Anyone compiling for aarch64-apple will be using 1.49+ anyway.

Reply:
Is the bump of 1.33 still necessary?

Reply:
Is dumping 1.33 necessary for this PR ? thanks and sorry to be a pain :)

Reply:
libc, which is needed on the M1 for 'cp' to link, has a minimum version of 1.33 for all dependencies https://crates.io/crates/libc